code/src/App.js

```diff
@@ -1,9 +1,96 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { AwFrequency } from './AwFrequency'
+import { WantThemes } from './WantThemes'
+import { ThemeFrequency } from './ThemeFrequency'
+import { ThemeSuggestion } from './ThemeSuggestion'
 
+// Set of arrays
+const awFrequencies = ['Every week', 'Every two weeks', 'Every month']
+const themeFrequencies = ['Every afterwork', 'Every second afterwoks', 'once a year']
+const themeAwChoices = ['YES', 'NO']
+
+// Hooks that I am using in the App
 export const App = () => {
+  const [awFrequency, setAwFrequency] = useState(awFrequencies[0])
+  const [themeAw, setThemeAw] = useState('')
```

Comment:
What about making this to a bool instead? Just because the answers is yes or no only